crates/driver/src/infra/solver/dto/solution.rs

```diff
@@ -224,6 +227,7 @@ impl Solutions {
                                 flashloan_hints.get(&uid).cloned()?,
                             ))
                         }).collect()),
+                    solution.wrappers.clone().into_iter().map(|w| WrapperCall(eth::Address(w.0), w.1)).collect()
```

Comment:
nit: Instead of cloning the whole vector only to move out all the values immediately afterwards you can just iterate over all the items and clone each one individually. That way we don't have to trust that the optimizer ellides the allocation of the temporary object and it better conveys the intent here. This is especially true if you were to `.filter()` the items of the vector.
```suggestion
                    solution.wrappers.iter().cloned().map(|w| WrapperCall(eth::Address(w.0), w.1)).collect()
```